src/http_server.cc

```diff
@@ -4688,12 +4697,40 @@ HTTPAPIServer::Create(
       server, trace_manager, shm_manager, port, reuse_port, address,
       header_forward_pattern, thread_cnt, restricted_features));
 
-  const std::string addr = address + ":" + std::to_string(port);
-  LOG_INFO << "Started HTTPService at " << addr;
 
   return nullptr;
 }
 
+
+bool
+HTTPAPIServer::Create_Wrapper(
+    std::shared_ptr<TRITONSERVER_Server>& server, UnorderedMapType& data,
+    std::unique_ptr<HTTPServer>* service,
+    const RestrictedFeatures& restricted_features)
+{
+  int port = get_value<int>(data, "port");
+  bool reuse_port = get_value<int>(data, "reuse_port");
+  std::string address = get_value<std::string>(data, "address");
+  std::string header_forward_pattern =
+      get_value<std::string>(data, "header_forward_pattern");
+  int thread_count = get_value<int>(data, "thread_count");
+
+  TRITONSERVER_Error* err = Create(
+      server,                     // shared_ptr<TRITONSERVER_Server>
+      nullptr, nullptr,           // TraceManager, SharedMemoryManager
+      port, reuse_port, address,  // port, reuse_port, address
+      header_forward_pattern,
+      thread_count,         // header_forward_pattern, thread_count
+      restricted_features,  // RestrictedFeatures
+      service);             // HTTPServer instance
```

Comment:
you don't need the comments when the variable names are descriptive, only for literals like the `nullptr`s for TraceManager/SHM Manager

Reply:
Removed comments for everything other than `TraceManager` and `SharedMemoryManager`